addons/l10n_eg/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,817 @@
+        <field name="parent_id" ref="tax_report_vat_return_sale_tax"/>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="2"/>
+    </record>
+
+    <record id="tax_report_vat_return_sale_tax_exempt" model="account.tax.report.line">
+        <field name="name">3. Exempt Sales (Tax)</field>
+        <field name="tag_name">3. Exempt Sales (Tax)</field>
+        <field name="code">EXM_SALE_T</field>
+        <field name="parent_id" ref="tax_report_vat_return_sale_tax"/>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="3"/>
+    </record>
+
+    <record id="tax_report_vat_return_sale_tax_4" model="account.tax.report.line">
+        <field name="name">4. Net Sales (Tax)</field>
+        <field name="formula">STD_SALE_T + EXM_SALE_T + ZERO_SALE_T</field>
+        <field name="parent_id" ref="tax_report_vat_return_sale_tax"/>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="4"/>
+    </record>
+
+    <record id="tax_report_vat_return_expense_base" model="account.tax.report.line">
+        <field name="name">VAT on Expenses and all other Inputs (Base)</field>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="3"/>
+    </record>
+
+    <record id="tax_report_vat_return_expense_base_fourteen" model="account.tax.report.line">
+        <field name="name">5. Standard Rated 14% Expenses (Base)</field>
+        <field name="tag_name">5. Standard Rated 14% Expenses (Base)</field>
+        <field name="code">STD_PUR_B</field>
+        <field name="parent_id" ref="tax_report_vat_return_expense_base"/>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="1"/>
+    </record>
+
+    <record id="tax_report_vat_return_expense_base_zero" model="account.tax.report.line">
+        <field name="name">6. Zero Rated (Base)</field>
+        <field name="tag_name">6. Zero Rated (Base)</field>
+        <field name="code">ZERO_PUR_B</field>
+        <field name="parent_id" ref="tax_report_vat_return_expense_base"/>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="2"/>
+    </record>
+
+    <record id="tax_report_vat_return_expense_base_exempt" model="account.tax.report.line">
+        <field name="name">7. Exempt Expenses (Base)</field>
+        <field name="tag_name">7. Exempt Expenses (Base)</field>
+        <field name="code">EXM_PUR_B</field>
+        <field name="parent_id" ref="tax_report_vat_return_expense_base"/>
+        <field name="report_id" ref="tax_report_vat_return"/>
+        <field name="sequence" eval="3"/>
+    </record>
+
+    <record id="tax_report_vat_return_expense_base_4" model="account.tax.report.line">
+        <field name="name">8. Net Expenses (Base)</field>
+        <field name="formula">STD_PUR_B + EXM_PUR_B + ZERO_PUR_B</field>
```

Comment:
@akr-odoo We have a line with net expenses which is the sum of the other lines, but in the meantime, we   also have the title with an amount.  (which seems to correspond, but...)